Sync/SyncStateMachine.swift

```diff
@@ -682,6 +718,24 @@ open class InitialWithLiveTokenAndInfo: BaseSyncStateWithInfo {
             log.debug("No cached meta/global found. Fetching fresh meta/global.")
         }
 
+        return deferMaybe(NeedsFreshMetaGlobal.fromState(self))
```

Comment:
The scratchpad's global and keys are `nil`ed out and pickled.

On the next time around, we'll take the alternative branch, because we no longer have any cached global.

Added a comment to this effect.